docs/next/tsconfig.json

```diff
@@ -18,6 +18,6 @@
     "downlevelIteration": true,
     "incremental": true
   },
-  "include": ["next-env.d.ts", "global.d.ts", "**/*.ts", "**/*.tsx"],
+  "include": ["next-env.d.ts", "global.d.ts", "**/*.ts", "**/*.tsx", "markdoc/config.js"],
```

Comment:
I don't see this file in this PR, is it in a different PR? Also, it doesn't seem quite right to include a `js` file here.